templates/web-deployment.yaml

```diff
@@ -46,15 +46,11 @@ spec:
           image: "{{ .Values.web.image.repository }}:{{ .Values.web.image.tag }}"
           imagePullPolicy: {{ .Values.web.image.pullPolicy }}
           env:
-            - name: TEMPORAL_GRPC_ENDPOINT
+            - name: TEMPORAL_ADDRESS
               value: "{{ include "temporal.fullname" . }}-frontend.{{ .Release.Namespace }}.svc:{{ .Values.server.frontend.service.port }}"
-          volumeMounts:
```

Comment:
why the `volumeMounts` has been deleted, it make the `web.config` in helm values no use.